lock/lock_test.go

```diff
@@ -0,0 +1,55 @@
+package lock
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/go-commons/logging"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBlockingAcquireLock(t *testing.T) {
+	t.Parallel()
+
+	lockString := "guardduty-blocking-acquire-lock-test"
```

Comment:
Use unique ID to ensure other tests running concurrently aren't competing for the same lock? Same for the table name too.